components/camera/platforms/jetson/data.go

```diff
@@ -0,0 +1,38 @@
+package jetsoncamera
+
+const (
+	// Jetson Devices
+	OrinAGX = "Jetson AGX Orin"
+	Unknown = "unknown"
+	// Daughterboards
+	// https://www.e-consystems.com/nvidia-cameras/jetson-agx-orin-cameras/full-hd-ar0234-color-global-shutter-camera.asp
+	ECAM = "e-CAM20_CUOAGX"
+	// Cameras
+	// https://www.e-consystems.com/camera-modules/ar0234-global-shutter-camera-module.asp
+	AR0234 = "AR0234CS"
+)
+
+type OSInformation struct {
+	Name   string // e.g. "linux"
+	Arch   string // e.g. "arm64"
+	Kernel string // e.g. "4.9.140-tegra"
+	Device string // e.g. "NVIDIA Jetson AGX Xavier"
+}
+
+type CameraInformation struct {
```

Comment:
From @bhaney 

Make sure to do a `make lint` before pushing the commit, because they will require you to write comments above any public struct or function